app/services/activitypub/synchronize_followers_service.rb

```diff
@@ -23,7 +23,7 @@ def call(account, partial_collection_url)
   private
 
   def remove_unexpected_local_followers!
-    @account.followers.local.where.not(id: @expected_followers.map(&:id)).each do |unexpected_follower|
+    @account.followers.local.where.not(id: @expected_followers.map(&:id)).reorder(nil).find_each do |unexpected_follower|
```

Comment:
There is a default order which comes in from the `followers` association, which causes an error when present at the time find_each runs (because find_each wants to replace the existing order with the batching order). The `reorder` here removes that prior to find_each being called. I think this is harmless because the order that these records are submitted into the service is not critical in this scenario.